occlum/occlum_workspace/.__occlum_status

```diff
@@ -0,0 +1 @@
+initialized
```

Comment:
occlum_workspace shouldn't be included.

Reply:
Generated during testing, has been deleted